src/components/Candidate.js

```diff
@@ -181,6 +181,10 @@ const Candidate = () => {
               <span className="candidate-prop-label">Contest:</span>
               {candidate.office}
             </p>
+            <p className="candidate-prop">
+              <span className="candidate-prop-label">PAC:</span>
+              {candidate.committee_name}
+            </p>
```

Comment:
not sure if the committee name is always defined, but in case it isn't we could guard against this with:
```suggestion
            {candiate.committee_name && (<p className="candidate-prop">
              <span className="candidate-prop-label">PAC:</span>
              {candidate.committee_name}
            </p>)}
```

But now that I'm thinking out loud, does a candidate ever not have a committee? 🤔 🤷‍♂️

Reply:
I don't think so, but I'd rather be safe than sorry. Great suggestion, thanks! 🥳